packages/components/src/custom-gradient-picker/index.tsx

```diff
@@ -100,30 +118,58 @@ const GradientTypePicker = ( { gradientAST, hasGradient, onChange } ) => {
 			onChange={ handleOnChange }
 			options={ GRADIENT_OPTIONS }
 			size="__unstable-large"
-			value={ hasGradient && type }
+			// NTS: Small runtime change. Previously, if `type` was undefined (which I don't think should ever happen) this check would return `false`, which would be an invalid value.
+			value={ hasGradient ? type : undefined }
```

Comment:
Small runtime change. Previously, if `type` was undefined (which I don't think should ever happen) this check would return `false`, which would be an invalid value.